include/range/v3/view/drop_exactly.hpp

```diff
@@ -51,10 +51,18 @@ namespace ranges
             difference_type_ n_;
 
             // RandomAccessRange == true
+            template<typename BaseRng = Rng,
+                CONCEPT_REQUIRES_(RandomAccessRange<BaseRng const>())>
```

Comment:
....since `iterator_t<T>` is `decltype(begin(declval<T&>()))` and `sentinel_t<T>` is `decltype(end(declval<T&>()))`.